config/fcos/v1_3_exp/translate_test.go

```diff
@@ -0,0 +1,1153 @@
+// Copyright 2020 Red Hat, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.)
+
+package v1_3_exp
+
+import (
+	"testing"
+
+	baseutil "github.com/coreos/fcct/base/util"
+	base "github.com/coreos/fcct/base/v0_3"
+	"github.com/coreos/fcct/config/common"
+	"github.com/coreos/fcct/translate"
+
+	"github.com/coreos/ignition/v2/config/util"
+	"github.com/coreos/ignition/v2/config/v3_2/types"
+	"github.com/coreos/vcontext/path"
+	"github.com/coreos/vcontext/report"
+	"github.com/stretchr/testify/assert"
+)
+
+// Most of this is covered by the Ignition translator generic tests, so just test the custom bits
+
+// TestTranslateBootDevice tests translating the FCC boot_device section.
+func TestTranslateBootDevice(t *testing.T) {
+	tests := []struct {
+		in         Config
+		out        types.Config
+		exceptions []translate.Translation
```

Comment:
Hmm, should this be called e.g. `translations` instead?

Reply:
No, this is the `exceptions` argument to `VerifyTranslations()`.  It just so happens that every translation here is an exception (except for two of them, oops).